lib/jsonapi-response.js

```diff
@@ -60,58 +59,90 @@ function getRelationships (data) {
     relationships.places = creationDetails.places;
   }
 
+  if (relatedItems) {
+    relationships.objects = {};
+    relationships.documents = {};
+    relationships.objects.data = relatedItems.relatedObjects.map(returnRelationship);
+    relationships.documents.data = relatedItems.relatedDocuments.map(returnRelationship);
+  }
+
   if (Object.keys(relationships).length) {
     return relationships;
   }
 }
 
-function getIncluded (data, relationships, config) {
+function returnRelationship (el) {
+  return {
+    type: el.type,
+    id: el.id
+  };
+}
+
+function getIncluded (data, relationships, config, relatedItems) {
   if (relationships) {
-    const included = [];
+    var included = [];
```

Comment:
Based on your previous suggestion not to use const for arrays/objects that are being added to, I've changed any I've noticed from before that were const